scrapy/templates/project/module/settings.py.tmpl

```diff
@@ -16,75 +16,75 @@ NEWSPIDER_MODULE = '$project_name.spiders'
 
 
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
-#USER_AGENT = '$project_name (+http://www.yourdomain.com)'
+# USER_AGENT = '$project_name (+http://www.yourdomain.com)'
```

Comment:
I believe this is on purpose. That space is the difference between a regular comment and commented-out code that you are supposed to comment out if you wish.

Reply:
I hadn't thought of that, it makes sense. But on the other hand, I'm not sure having automatically generated files being non-PEP8 compliant is a good idea.
I think the fact that settings are uppercase should be enough to tell them from regular comments IMHO.
Let's see what others have to say about it: this PR comes from a personal "need", I end up modifying the settings file in every new project I work on.